en/docs/references/pricing.md

```diff
@@ -0,0 +1,227 @@
+# Pricing
+
+## Introduction
+
+Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. You can use Choreo by paying only for your usage: nothing more, nothing less. Choreo has payment plans for all types of users. You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you use more than 5,000 [steps](#what-is-a-step) per month. If you plan to use Choreo significantly, you can also opt for an Enterprise plan and enjoy special discount rates.
+
+## Plans
+
+|               | Free                   |  Pay As You Go    | Enterprise     |
+|---------------|------------------------|-------------------|----------------|
+| Cost Per Step | 5,000 steps per month *|$5 per 1,000 steps <br/> <ul><li>First 5,000 steps per month are free</li></ul>| Contact Us For Discounted Pricing <br/><ul><li> Volume Discounts</li><li> Committed-use Discounts</li><li> Support Channels and Support SLA</li></ul>|
+
+ \* Limited to a maximum of 3 components
+
+
+
+## Choreo pricing FAQ
+
+### How does Choreo pricing work?
+
+Choreo pricing is based on the number of [steps](#what-is-a-step) in your integration solution. Choreo offers three payment plans: Free, Pay As You Go, and Enterprise.
+
+#### Free 
+The Choreo free tier allows you to execute 5,000 [steps](#what-is-a-step) per month, limited to a maximum of 3 components. If you exceed the free tier, your application cannot make any additional requests to Choreo. 
```

Comment:
``` suggestion
The Choreo free tier allows you to execute 5,000 [steps](#what-is-a-step) per month for a maximum of 3 components. If you exceed the free tier, your application cannot make any additional requests to Choreo.
```

Reply:
Same as above

Reply:
The Google Developer Guide which you have linked states the following:

"Numbers as words
In general, spell out the following:

Numbers from zero through nine, except as noted in Numbers as numerals."

Reply:
same as above